mixer/cmd/bundles.go

```diff
@@ -198,12 +201,74 @@ any errors are encountered earlier on.`,
 	},
 }
 
+// Bundle validate command ('mixer bundle validate')
+type bundleValidateCmdFlags struct {
+	allLocal      bool
+	strictnessLvl string
+}
+
+var bundleValidateFlags bundleValidateCmdFlags
+
+var bundleValidateCmd = &cobra.Command{
+	Use:   "validate <bundle> [<bundle>...]",
+	Short: "Validate local bundle definition files",
+	Long: `Checks local bundle definition files for validity as a separate step. While
+'mixer bundle edit' will automatically validate bundles as you edit them, it can
+be useful to manually run this validation step on bundles you have previously
+written, or have edited using a different tool. Only local bundle files are
+checked; upstream bundles are trusted as valid. Valid bundles yield no output.
+Any invalid bundles will yield a non-zero return code.
```

Comment:
Optional: I felt like this is a bit too much about 'edit'. I think you can shave some words by ignoring that fact (there's a balance: we want long descriptions to be as short as they can :). Describe what the command does (no need to say it is a separate step), and then mention that "mixer bundle edit" automatically calls the same checks.

Reply:
Optional: some user-consumable description of what is being checked would be nice.

Reply:
Fixed in new push.